internal/pkg/manifest/svc.go

```diff
@@ -27,7 +28,7 @@ const (
 	// WorkerServiceType is a worker service that manages the consumption of messages.
 	WorkerServiceType = "Worker Service"
 	// StaticSiteType is a static site service that manages static assets.
-	StaticSiteType = "Static Site"
+	StaticSiteType = "Static Site Service"
```

Comment:
Hmmmmm :thinkspin: I kinda liked the previous version. Can you check with @genbit about our preference?

Reply:
We check if it ends with "Service" to determine if it is a service or job in `config` pkg. Do you think we can make `Static Site` an exception?

Reply:
🤦 I feel like we could have something like this:
```go
package manifest

type Type string
const (
   StaticSiteType = "Static Site"
)

var serviceTypes = []Type{StaticSiteType, WorkerServiceType, ...}

func (t Type) IsService() bool {
  for _, valid range := serviceTypes {
     if valid == t {
        return true
     }
  }
  return false
}
```
what do you think?

Reply:
next pr i'll move types to a new pkg